sklearn/semi_supervised/_label_propagation.py

```diff
@@ -384,6 +387,18 @@ class LabelPropagation(BaseLabelPropagation):
 
     n_iter_ : int
         Number of iterations run.
+    LabelPropagation(...)
```

Comment:
This line corresponds to the `Examples` section. It is the output of the `fit` method.